src/components/RadioButton/CheckBox.tsx

```diff
@@ -1,15 +1,15 @@
 import React from "react";
 import styles from "./RadioButton.module.scss";
 
-interface RadioButtonProps {
+interface CheckBoxProps {
```

Comment:
Directory RadioButton inside I see checkbox, this is inconsistency between name and usage, please move it in to Checkbox component folder

Reply:
By the way you need both radio and checkbox